### DISCUSSION TITLE
How does EGSnrc software compare to SRIM-2013? Are heating effects modelled?

### DISCUSSION THREAD

**Discussion by Hughes244** [nrc-cnrc/EGSnrc#903] [Questions] [answered]
How does this software compare to SRIM-2013? Is it possible to extrapolate heating effects from running proton simulations on a giving material? ie. what temperature increase is present with a given irradiation? 

Cheers! 

Tom.

**Comment by rtownson** [comment]
EGSnrc is not comparable because it does not model protons - only photons, electrons and positrons (but it does so particularly well).

**Reply by Hughes244** [reply]
Thank you!

**Comment by ftessier** [comment, marked as answer]
EGSnrc can tally energy deposition events, which you can extrapolate to local temperature rise with the material heat capacity. You could then use that as an input in a heat transport simulation software.

**Reply by Hughes244** [reply]
Are energy deposition events including both any transfer of energy to the electron cloud, and then collision events? Currently have been simulating the latter and effectively throwing those results at Abaqus for me. Neglects any electron cloud interaction though. 

Thanks for the reply!

**Reply by ftessier** [reply]
EGSnrc models interaction with atomic electrons, modelling transport of those, in turn, if they are ejected. It further accounts for shell vacancies thus created and the ensuing atomic relaxations. Bear in mind that the lowest energy of transported electrons in EGSnrc is 1 keV kinetic energy, below which the electron is discarded in the simulation and its energy is deposited on the spot.

**Reply by Hughes244** [reply]
Okay, thank you!

**Reply by rtownson** [reply]
For proton beams, some users will model the protons in a different code, save the resulting photons/electrons/positrons in IAEA phase-space format, and then simulate transport of those using EGSnrc.

**Reply by Hughes244** [reply]
Nice to know!
